### ISSUE TITLE
(refactor) End to End Tests Selectors

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#1056]
There are several tests that use non-specific selectors - this opens the test up to possible errors.  They appear like below:

```sh
W/element - more than one element found for locator By(css selector, [data-method="submit"]) - the first result will be used
```

This is usually because a developer tried to "submit" a modal using `FU.buttons.submit()`, and there are multiple submit buttons available to protractor.  The correct way to submit the modal would be `FU.modal.submit()`.

All of these instances need to change to ensure our tests continue to function correctly.  Here is a list of affected modules/testss:
 1. Aged Debtors
 2. Chart of Accounts Report Generation
 3. Cash Payments Registry
 4. Complex Vouchers

**Comment by jniles** [comment]
@mbayopanda, can you work on this next?  It should be really quick.

I'll fix the `Complex Voucher` one.  The rest you can adjust.

**Comment by jniles** [comment]
Closed in #1078.
